src/components/higlass/HiGlassSubscriber.js

```diff
@@ -13,8 +14,11 @@ import {
 } from '../../app/state/hooks';
 import { COMPONENT_COORDINATION_TYPES } from '../../app/state/coordination';
 
-const PIXI_BUNDLE_URL = 'https://unpkg.com/window-pixi@5.3.3/dist/pixi.min.js';
-const HIGLASS_BUNDLE_URL = 'https://unpkg.com/higlass@1.11.4/dist/hglib.min.js';
+const PIXI_BUNDLE_VERSION = packageJson.dependencies['window-pixi'];
+const HIGLASS_BUNDLE_VERSION = packageJson.dependencies.higlass;
+const BUNDLE_FILE_EXT = process.env.NODE_ENV === 'development' ? 'js' : 'min.js';
```

Comment:
How do we have access to this variable?

Reply:
Which variable? process is available due to node/webpack

Reply:
Yes, that's what I was wondering about.  Webpack adds that variable to the browser bundle?

Reply:
Yes I believe so

Reply:
https://webpack.js.org/plugins/define-plugin/

Reply:
Hmmm, ok.  Maybe worth a comment then.  If ever move out of webpack, we'll need to remember to add this variable.

Reply:
Rollup has it too https://github.com/rollup/plugins/tree/master/packages/replace